sections/exchange/hooks/useExchange.tsx

```diff
@@ -114,7 +114,14 @@ const useExchange = ({
 }: ExchangeCardProps) => {
 	const { t } = useTranslation();
 	const { monitorTransaction } = TransactionNotifier.useContainer();
-	const { hasNone: hasNoL2Gas } = L2Gas.useContainer();
+	// const { hasNone: hasNoL2Gas } = L2Gas.useContainer();
```

Comment:
Let's not put commented code in the repo. Remove anything we have to remove, we can always revert in the future.